lib/utils/docker.coffee

```diff
@@ -48,6 +48,12 @@ exports.appendOptions = (opts) ->
 			alias: 't'
 		},
 		{
+			signature: 'arg'
+			parameter: 'arg'
+			description: 'Set a build-time variable (eg. "-e \'ARG=value\'"). Can be specified multiple times.'
+			alias: 'e'
```

Comment:
How about `--arg/-A`? Not sure how Capitano handles option case though, will have to test.